server/lightshows/christmas.py

```diff
@@ -17,7 +17,7 @@ def init_parameters(self):
                            "chunk_blendover": 5,
                            "whole_blendover": 5}
 
-    def set_parameter(self, param_name: str, value):
+    def set_parameter(self, param_name: str, value, **kwargs):
```

Comment:
How does this help syncing the brightness?

Reply:
Not really, but I had to add this in the beginning to get it running. I will have a closer look this evening.

Reply:
`Christmas` overrides the method `set_parameters` from `Lightshow` which is called from `apply_parameter`.

Without the fix running the christmas show leads to the following error:
```
...
  File ".../102shows/server/lightshows/templates/base.py", line 70, in __init__
    self.apply_parameter_set(parameters)
  File ".../102shows/server/lightshows/templates/base.py", line 193, in apply_parameter_set
    self.set_parameter(param_name, value=parameters[param_name], send_mqtt_update=False)
TypeError: set_parameter() got an unexpected keyword argument 'send_mqtt_update'
```

As this call seemed not to be how it is intended adding `**kwargs` was the simplest way to fix the Christmas show.

Reply:
I see. I added a call to the overridden method, for potential parameters that the overriding method does not cover.
I will fix this properly (using `Lightshow.p`) next week